tests/selenium/test_basic_operations.py

```diff
@@ -0,0 +1,323 @@
+
+    with test.table('table') as target:
+        target.cell.click(249, 0)
+        test.send_keys('abc' + Keys.UP)
+
+        assert target.cell.get_text(249, 0) == 'abc'
+        assert target.cell.is_focused(248, 0)
+
+
+# https://github.com/plotly/dash-table/issues/141
+def test_tbst009_active_focused_arrow_down(test):
+    test.start_server(get_app())
+
+    with test.table('table') as target:
+        target.cell.click(249, 0)
+        test.send_keys('abc' + Keys.DOWN)
+
+        assert target.cell.get_text(249, 0) == 'abc'
+        assert target.cell.is_focused(249, 0)
+
+
+def test_tbst010_active_with_dblclick(test):
+    test.start_server(get_app())
+
+    with test.table('table') as target:
+        target.cell.double_click(0, 0)
+        assert target.cell.is_active(0, 0)
+        assert target.cell.get_text(0, 0) == test.get_selected_text()
+
+
+def test_tbst011_delete_row(test):
+    test.start_server(get_app())
+
+    with test.table('table') as target:
+        text00 = target.cell.get_text(0, 0)
+        text01 = target.cell.get_text(1, 0)
+        target.row.delete(0)
+
+        assert target.cell.get_text(0, 0) == text01
+
+
+def test_tbst012_delete_sorted_row(test):
+    test.start_server(get_app())
+
+    with test.table('table') as target:
+        target.column.sort(0, rawDf.columns[0]) # None -> ASC
+        target.column.sort(0, rawDf.columns[0]) # ASC -> DESC
+
+        text00 = target.cell.get_text(0, 0)
+        text01 = target.cell.get_text(1, 0)
+        target.row.delete(0)
+
+        assert target.cell.get_text(0, 0) == text01
+
+
+def test_tbst013_select_row(test):
+    test.start_server(get_app())
+
+    with test.table('table') as target:
+        target.row.select(0)
+
+        assert target.row.is_selected(0)
+
+
+def test_tbst014_selected_sorted_row(test):
+    test.start_server(get_app())
+
+    with test.table('table') as target:
+        target.column.sort(0, rawDf.columns[0]) # None -> ASC
+        target.column.sort(0, rawDf.columns[0]) # ASC -> DESC
+        target.row.select(0)
+
+        assert target.row.is_selected(0)
+
+
+def test_tbst015_selected_row_respects_sort(test):
+    test.start_server(get_app())
+
+    with test.table('table') as target:
+        target.row.select(0)
+
+        assert target.row.is_selected(0)
+
+        target.column.sort(0, rawDf.columns[0]) # None -> ASC
+        target.column.sort(0, rawDf.columns[0]) # ASC -> DESC
+
+        assert not target.row.is_selected(0)
+
+        target.column.sort(0, rawDf.columns[0]) # DESC -> None
+
+        assert target.row.is_selected(0)
+
+
+def test_tbst016_delete_cell(test):
+    test.start_server(get_app())
+
+    with test.table('table') as target:
+        target.cell.click(0, 1)
+        test.send_keys(Keys.BACKSPACE)
+        test.send_keys(Keys.ENTER)
+
+        assert target.cell.get_text(0, 1) == ''
+
+
+# # https://github.com/plotly/dash-table/issues/700
+# def test_tbst017_delete_cell_updates_while_selected(test):
```

Comment:
Are you intending to fix that bug in this PR, or just putting the test code here for re-enabling whenever that bug gets fixed? You can also do
```py
@pytest.mark.skip(reason="https://github.com/plotly/dash-table/issues/700")
def test_tbst017_delete_cell_updates_while_selected(test):
    ...
```

Reply:
Nice, will update to use the decorator

Reply:
https://github.com/plotly/dash-table/pull/696/commits/ea38a8604e6ab7540b8e13eb2e8c5e5401a0f059